swarm/api/api.go

```diff
@@ -504,15 +504,16 @@ func (self *Api) AddFile(mhash, path, fname string, content []byte, nameresolver
 
 }
 
-func (self *Api) RemoveFile(mhash, path, fname string, nameresolver bool) (string, error) {
+// RemoveFile - remove's a file's entry in a manifest
```

Comment:
removes

Reply:
file entry

Reply:
```
RemoveFile removes a file entry in a manifest.
```

From GoDoc - just use english sentences and start the sentence with the identifier being described.